### ISSUE TITLE
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 

### ISSUE THREAD

**Issue by danewallace** [oznu/homebridge-config-ui-x#852]
**Describe Your Problem:**
After upgrading Homebridge and Config UI to latest releases I am seeing the following output in the log every 10-15 seconds are so:
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 

Homebridge loads, and plugins/devices work as expected, but the output completely clutters the log and makes it hard to parse actually helpful info. 

For context, Homebridge is running on a macOS VM hosted on a macOS system. The VM's network connection is bridged so it has its own IP address. The VM's system preferences are configured for its ETH interface to be configured, and Wi-Fi is indeed not present in the VM. The Mac hosting the VM has Wi-Fi configured and has an IP without issue. This issue did not appear until the most recent update.

**Logs:**
[9/19/2020, 11:35:00 AM] Homebridge is running on port 51750.
[9/19/2020, 11:35:08 AM] [Living Room TV] Failed to pair! Make sure TV is online and you click "Allow" on the popup
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
[9/19/2020, 11:37:39 AM] [homebridge] Homebridge restart request received
[9/19/2020, 11:37:39 AM] [homebridge] UI / Bridge settings have not changed; only restarting Homebridge process
[9/19/2020, 11:37:39 AM] [HB Supervisor] Sending SIGTERM to Homebridge
[9/19/2020, 11:37:39 AM] Got SIGTERM, shutting down Homebridge...
[9/19/2020, 11:37:44 AM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[9/19/2020, 11:37:49 AM] [HB Supervisor] Restarting Homebridge...
[9/19/2020, 11:37:49 AM] [HB Supervisor] Starting Homebridge with extra flags: -I
[9/19/2020, 11:37:49 AM] [HB Supervisor] Started Homebridge v1.2.2 with PID: 741
[9/19/2020, 11:37:49 AM] Loaded config.json with 2 accessories and 5 platforms.
[9/19/2020, 11:37:49 AM] ---
[9/19/2020, 11:37:50 AM] Loaded plugin: homebridge-config-ui-x@4.27.1
[9/19/2020, 11:37:50 AM] Registering platform 'homebridge-config-ui-x.config'
[9/19/2020, 11:37:50 AM] ---
[9/19/2020, 11:37:50 AM] Loaded plugin: homebridge-dummy@0.4.1
[9/19/2020, 11:37:50 AM] Registering accessory 'homebridge-dummy.DummySwitch'
[9/19/2020, 11:37:50 AM] ---
[9/19/2020, 11:37:50 AM] Loaded plugin: homebridge-frigidaire@1.0.7
[9/19/2020, 11:37:50 AM] Registering platform 'homebridge-frigidaire.Frigidaire'
[9/19/2020, 11:37:50 AM] ---
[9/19/2020, 11:37:50 AM] Loaded plugin: homebridge-platform-wemo@1.6.1
[9/19/2020, 11:37:50 AM] Registering platform 'homebridge-platform-wemo.BelkinWeMo'
[9/19/2020, 11:37:50 AM] ---
[9/19/2020, 11:37:51 AM] Loaded plugin: homebridge-ring@9.9.0
[9/19/2020, 11:37:51 AM] Registering platform 'homebridge-ring.Ring'
[9/19/2020, 11:37:51 AM] ---
[9/19/2020, 11:37:52 AM] Loaded plugin: homebridge-samsung-tizen@4.3.3
[9/19/2020, 11:37:52 AM] Registering platform 'homebridge-samsung-tizen.SamsungTizen'
[9/19/2020, 11:37:52 AM] ---
[9/19/2020, 11:37:52 AM] Loading 5 platforms...
[9/19/2020, 11:37:52 AM] [WeMo Platform] Initializing BelkinWeMo platform...
[9/19/2020, 11:37:52 AM] [SamsungTizen] Initializing SamsungTizen platform...
[9/19/2020, 11:37:52 AM] [Frigidaire AC Units] Initializing Frigidaire platform...
[9/19/2020, 11:37:52 AM] [homebridge] Initializing config platform...
[9/19/2020, 11:37:52 AM] [homebridge] Running in Service Mode
[9/19/2020, 11:37:52 AM] [Ring] Initializing Ring platform...
[9/19/2020, 11:37:52 AM] Loading 2 accessories...
[9/19/2020, 11:37:52 AM] [Arrived Home] Initializing DummySwitch accessory...
[9/19/2020, 11:37:52 AM] [Left Home] Initializing DummySwitch accessory...
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory e1255fe1-5bc6-4ba7-a2a2-059af0d3f1f5 Ring Alarm
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory c2d1dbd7-d5aa-480c-ae54-5ff4da1480d2 Base Station
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 1f2c0a7c-0d20-4b2a-814c-5dc7970f2921 Entryway Keypad
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 412fbcfd-1435-405e-9817-145442ee254f Front Balcony Door
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 487f417d-c83a-437e-9db1-4f0ebf5295eb Smoke/CO Listener Dining Room
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory cdee9311-2547-4823-8acf-97bae7ff1b01 Back Door
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 1c9a782b-5790-46fb-b7a1-0a6dd735ed13 Front Stairs
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 43005ec7-c63b-4668-8d03-a3059d6ecebf Front Door
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 65f901e1-b1e3-4fdd-9530-3b678c548f74 Garage
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory fd03bc03-2d7f-45e0-899d-b0e49cec9cbc Laundry Room
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 7e13ceda-1040-4c03-b771-0acf1f853973 Front Stairs
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory 8bab8721-5158-4ba6-a95f-7393ca616051 Living Room
[9/19/2020, 11:37:52 AM] [Ring] Configuring cached accessory bab0cd79-11de-4098-b928-1a29fe1c0a55 Master Bedroom
[9/19/2020, 11:37:52 AM] Living Room TV 9661 is running on port 49371.
[9/19/2020, 11:37:52 AM] Please add [Living Room TV 9661] manually in Home app. Setup Code: 381-25-022
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
[9/19/2020, 11:37:53 AM] [Ring] Creating location socket.io connection
[9/19/2020, 11:37:54 AM] [Ring] Ring connected to socket.io server
[9/19/2020, 11:37:54 AM] [Ring] Configuring 4 cameras and 18 devices for location "Los Angeles" - locationId: 7f39dc6c-eefd-4e2a-85ac-d7d72336e2c9
[9/19/2020, 11:37:54 AM] [Ring] Hidden accessory 3818a89f-922b-4c04-b3ef-3bb84c4704b6 adapter.zigbee ZigBee onboard adapter
[9/19/2020, 11:37:54 AM] [Ring] Hidden accessory 58a5ee47-27ae-4821-927a-f7819ec9b921 access-code.vault Code Vault
[9/19/2020, 11:37:54 AM] [Ring] Hidden accessory 1d75ee1a-018b-4290-a87c-928602748714 access-code Security Access Code
[9/19/2020, 11:37:54 AM] [Ring] Hidden accessory ea7520fb-61bd-451d-ade0-6d183c92fd4e adapter.zwave Z-Wave Adapter
[9/19/2020, 11:37:54 AM] [Ring] Hidden accessory 57e31289-89b0-42bb-8e19-f1482225f96d range-extender.zwave Living Room
Got Devices
[
  {
    MAKE: 'Frigidaire',
    MODEL: 'FGRC1044T101',
    APPLIANCE_INFO_ID: 2,
    FIRMWARE_VERSION: null,
    TIME_ZONE: null,
    NIU_VERSION: 'v4.9-6-3',
    APPLIANCE_ID: 132974,
    APPLIANCE_TYPE_ID: 3,
    LABEL: 'Living Room AC',
    MAC_ADDRESS: '6014B351FACE',
    SERIAL: 'KK75245090',
    APPLIANCE_TYPE_DESC: 'Air Conditioner',
    NOTIFICATIONS_ENABLED: 1
  },
  {
    MAKE: 'Frigidaire',
    MODEL: 'FGRC064WA100',
    APPLIANCE_INFO_ID: 2,
    FIRMWARE_VERSION: null,
    TIME_ZONE: null,
    NIU_VERSION: 'v4.8-5-2',
    APPLIANCE_ID: 180401,
    APPLIANCE_TYPE_ID: 3,
    LABEL: 'Master Bedroom AC',
    MAC_ADDRESS: 'F8A2D611176A',
    SERIAL: 'KK90223031',
    APPLIANCE_TYPE_DESC: 'Air Conditioner',
    NOTIFICATIONS_ENABLED: 1
  }
]
craeting accessory for AC unit labeled : Living Room AC
craeting accessory for AC unit labeled : Master Bedroom AC
[9/19/2020, 11:37:55 AM] Initializing platform accessory 'Living Room AC'...
[9/19/2020, 11:37:55 AM] Initializing platform accessory 'Master Bedroom AC'...
Setup Payload:
X-HM://00243MPJY75PI
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 381-25-022 │     
    └────────────┘     
                       
[9/19/2020, 11:37:55 AM] Homebridge is running on port 51750.
[9/19/2020, 11:38:04 AM] [Living Room TV] Failed to pair! Make sure TV is online and you click "Allow" on the popup
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
[9/19/2020, 11:38:52 AM] [WeMo Platform] Online: Pink Entler [94103E38C2A8]
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 
CIAO WARN: While checking networksetup for en0 encountered an unknown output: All Wi-Fi network services are disabled.; 

**Homebridge Config:**
{
    "bridge": {
        "name": "Homebridge",
        "username": "0E:09:E3:B2:E6:92",
        "port": 51750,
        "pin": "381-25-022"
    },
    "accessories": [
        {
            "accessory": "DummySwitch",
            "name": "Arrived Home"
        },
        {
            "accessory": "DummySwitch",
            "name": "Left Home"
        }
    ],
    "platforms": [
        {
            "platform": "BelkinWeMo",
            "name": "WeMo Platform",
            "ignoredDevices": []
        },
        {
            "name": "SamsungTizen",
            "devices": [
                {
                    "name": "Living Room TV",
                    "ip": "192.168.4.97",
                    "mac": "24:fc:e5:92:ea:34"
                }
            ],
            "platform": "SamsungTizen"
        },
        {
            "platform": "Frigidaire",
            "name": "Frigidaire AC Units",
            "username": "omited@icloud.com",
            "password": "omitted"
        },
        {
            "name": "homebridge",
            "port": 8580,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "proxyHost": "192.168.1.12:8580",
            "platform": "config"
        },
        {
            "refreshToken": "omitted",
            "platform": "Ring"
        }
    ]
}

**Screenshots:**

**Environment:**

Timezone | GMT-0700
-- | --
macOS Catalina
homebridge
192.168.1.12
fe80::1046:9ebf:982f:485f

danewallace
/Users/danewallace/.homebridge
/Users/danewallace/.homebridge/config.json
Yes

* **Node.js Version**: v12.18.3
* **NPM Version**: v6.14.6
* **Homebridge Version**: Version 1.2.2
* **Homebridge Config UI X Version**:  v4.27.1
* **Operating System**: macOS 
* **Process Supervisor**:  hb-service

**Comment by oznu** [comment]
Please comment on the issue here: https://github.com/homebridge/homebridge/issues/2681

Be sure to follow the instructions in the top comment and provide the information requested.

**Comment by Supereg** [comment]
fixed with 1.2.3
